### DISCUSSION TITLE
How to plot `plot_series` results with Plotly dash

### DISCUSSION THREAD

**Discussion by matsuobasho** [Nixtla/mlforecast#352] [Q&A]
I'm trying to plot the output of `plot_series` in a Plotly Dash dashboard.  From the first print statement, it appears that `fig_html` is actually a string.  So I did some searching and that's why I have the `fig = pio.from_html(fig_html)` line.  However, it still doesn't work - my console never gets to the `print(fig)` line.

```
@app.callback(
    Output('results-comparison', 'figure'),
    Input('prediction', 'data'),
    Input('test-data', 'data'),
)
def plot_results(y_hat, test):
    y_hat = pd.DataFrame(y_hat)
    test = pd.DataFrame(test
    fig_html = plot_series(test[['ds', 'y', 'unique_id']], y_hat[['ds', 'y', 'LGBMRegressor']], engine='plotly')
    print(fig_html)
    fig = pio.from_html(fig_html)
    print(fig)
    print(type(fig))
    return fig
```

**Comment by jmoralez** [comment]
Hey. I'm not very familiar with plotly dash, but [internally](https://github.com/Nixtla/utilsforecast/blob/fb00c822f0330b451d6f54661750e0be1e132c5e/utilsforecast/plotting.py#L213) we use [make_subplots](https://plotly.com/python-api-reference/generated/plotly.subplots.make_subplots.html) which returns a [plotly Figure](https://plotly.com/python-api-reference/generated/plotly.graph_objects.Figure.html). Have you tried just `return plot_series(...)`?

**Comment by matsuobasho** [comment]
@jmoralez thanks for the quick response!  I took a more detailed look and am pretty sure now that the issue is with the Plotly Dash code and has nothing to do with mlforecast.  Will wait a few days to investigate more and then close this if I confirm this.
